app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
```

Comment:
Do you need this one? Have you guys changed anything with the devise login?

Reply:
Yes, we added two custom fields: first name, last name.

Reply:
Do you want the user to fill in its name and last name everytime they login?

Reply:
Ah, sorry, confused this one with our Sign Up page. You're correct, deleted now.